docs/pages/kubernetes-access/helm/reference.mdx

```diff
@@ -864,6 +864,29 @@ A list of `initContainers` which will be run before the main Teleport container
   </TabItem>
 </Tabs>
 
+## `postStart`
```

Comment:
Do you mind adding the same kind of table you added in https://github.com/gravitational/teleport/pull/7167/files:

```
## `extraEnv`

| Type | Default value | Can be used in `custom` mode? |
| - | - | - |
| `list` | `[]` | ✅  |
```